RELEASE_NOTES.md

```diff
@@ -1,27 +1,114 @@
 
-# UNRELEASED
-
-
-To expand upon later:
-
-- Improved ohai information gathering on on-English systems
-- Performance improvements (provide some benchmarks)
-- Package data in ohai for macOS
-- rhsm_register: reduced load on the Satellite server
-- new chef_client_launchd resource
-- osx_profile lets you remove profiles on macOS 11
-- chef_client_cron: Added nice property
-- Improved output in knife config list-profiles and new shorter knife config commands
-- New packages: macOS 11, s390x RHEL 7, s390x SLES 12
-- New chef_client_trusted_certificate
-- New exit code 43 when the client config has errors. Thanks @NaomiReeves
-- Mount: Improved idempotency on Linux hosts
-- chef_client_systemd_timer: Add cpu_quote property
-- Resolved knife ssh freezing on Windows nodes
-- launched: Improved validation for the values allowed in the nice property
-- windows_ad_join: added reboot_delay property
-- openssl to 1.0.2w
+# What's New in 16.5
+
+## Performance Improvements
+
+We continue to reduce the size of the Chef Infra Client install and optimize the performance of the client. With Chef Infra Client 16.5 we've greatly reduced the startup time of the `chef-client` process. Startup times on macOS, Linux, and Windows hosts are now approximately 2x faster than the 16.4 release.
+
+## CLI Improvements
+
+- The client license acceptance logic has been improved to provide helpful error messages when an incorrect value is passed and to accept license values in any text case.
+- A new `chef-client` process exit code of 43 has been added to signal that an invalid configuration was specified. Thanks [@NaomiReeves](https://github.com/NaomiReeves)!
+- The `knife ssh` command no longer hangs when connecting to Windows nodes over SSH
+- The `knife config` commands have been renamed to make them shorter and table output has been improved:
+  - knife config get-profile -> knife config use
+  - knife config use-profile [NAME] -> knife config use [NAME]
+  - knife config list-profiles -> knife config list
+  - knife config get -> knife config show
+
+## New Resources
+
+### chef_client_launchd
+
+The `chef_client_launchd` resource allows you to configure Chef Infra Client to run as a global launchd daemon on macOS hosts. This resource mirrors the configuration of other `chef_client_*` resources and allows for simple out of the box configuration of the daemon, while also providing advanced tunables. If you've used the `chef-client` cookbook in the past, you'll notice a number of improvements in the new resource including configuration update handling, splay times support, process nice level support, and an out of the box configuration of low IO priority execution. In order to handle restarting the Chef Infra Client launchd daemon when configuration changes occur the resource also installs a new `com.chef.restarter` daemon. This daemon watches for daemon configuration changes and gracefully handles the restart to ensure the client process continues to run.
```

Comment:
For the phrase `process nice level support,` would this make more sense "`nice` process level support"  or just "`nice` level support"?

Reply:
Updated to remove process